kmq.js

```diff
@@ -11,34 +11,86 @@ const db = new sqlite3.Database('./main.db', (err) => {
         return;
     }
 });
-var scoreboard = {};
+var scoreboard = [];
 var currentSong = null;
 var currentArtist = null;
+var currentSongLink = null;
 var gameInSession = false;
+
+const songInfoMessage = (message, currentSong, currentArtist, currentSongLink, isQuit) => {
```

Comment:
method name doesn't describe what it actually does, aka send the message

Reply:
is sendSongMessage better?

Reply:
sure